src/main/scala/beam/utils/BeamVehicleUtils.scala

```diff
@@ -135,6 +148,7 @@ object BeamVehicleUtils {
       seatingCapacity = 4,
       standingRoomCapacity = 0,
       lengthInMeter = 4.5,
+      curbWeightInKg = 2000,
```

Comment:
same here, using value from config would be better

Reply:
That's a fallback for missing types. I think it's better not to blow up the config. One could always create a missing type in vehicleTypes.csv.